spec/acceptance/suites/README.md

```diff
@@ -0,0 +1,121 @@
+# simp-core Acceptance and integration tests
+
+In this directory there are tests that can be used to test SIMP in it's
+entirety, including all non ISO components.
+
+| Suite                    | Category    | Description                                                                              |
+| ------------------------ | ----------- | ---------------------------------------------------------------------------------------- |
+| default                  | Integration | Uses components from `Puppetfile.tracking`                                               |
+| install_from_rpm         | Release     | Uses RPMs from SIMP's packagecloud.io repos                                              |
+| install_from_tar         | Release     | Uses RPMs using the tarball from the ISO build process                                   |
+| install_from_core_module | Release     | Uses a module build from `metadata.json` and whatever dependencies from the Puppet Forge |
+| rpm_docker               | Build       | Used to build the SIMP ISO                                                               |
+
+
+## Overview
+
+In general, integration tests for SIMP follow the same general procedure:
+
+1. Spin up 3 vagrant boxes, one puppetserver (EL version changes based on
+   nodeset) and two clients (one EL6 and one EL7)
+2. Install, start and configure the puppetserver
+  * This is the step where most tests are different. See the subsections below
+   for more details
+3. Add all the agents to the puppetserver, and run `puppet agent -t`
+   until there are no more changes
+4. Modify the puppet environment and run further tests
+
+## Running the tests
+
+1. Set up your environment with a [ruby version manager](https://rvm.io/), [vagrant](https://www.vagrantup.com/), and [VirtualBox](https://www.virtualbox.org/)
+2. Install Ruby 2.1.9 (follow the guides from the link above)
+3. Install bundler: `gem install bundler`
+4. Install other dependencies: `gem install`
+5. Run the tests:
+
+```bash
+bundle exec rake beaker:suites[<suite>,<nodeset>]
+```
+
+There two nodesets per suite, `el7_server` and `el6_server`. They control the
+version of EL on the puppetserver.
+
+### default
+
+This test runs `puppet module build` in the root directory of `simp_core`,
+building a development version of the simp/simp_core metamodule. Then, it spins
+up a puppetserver and trys to install the development version of the module,
+using the Puppet Forge as a source for all modules.
+
+However, if there are unreleased components referenced in the `metadata.json`,
+the `puppet module install` for the module will fail.
```

Comment:
No.  This does not make sense.

Also, this does not run 'simp config' and 'simp bootstrap'.